### ISSUE TITLE
Kubernetes configs docs broken

### ISSUE THREAD

**Issue by jash2105** [wekan/wekan#5522]
## Enhancement: Improve Kubernetes Documentation and Helm Chart for Wekan

Hello,

I noticed that the Kubernetes link in the developer documentation for Wekan is currently broken (the link is: [Kubernetes Documentation](https://github.com/wekan/wekan/tree/main/helm/wekan)). Upon further investigation, I explored the Helm chart ([Helm Chart Repository](https://github.com/wekan/charts)) and observed that the documentation and configurations may not be fully production-ready or polished.

Given this, I am in the process of setting up Wekan in a scalable environment and would love to contribute my work to enhance the Helm chart and documentation. My goal is to make the deployment process seamless for users, ensuring it works across multi-cloud environments and various Kubernetes distributions.

Here’s a brief overview of what I plan to contribute:
- Improved documentation and configurations for a production-ready deployment.
- Scalable configurations, including HPA (Horizontal Pod Autoscaler) and advanced ingress settings ( than what we have right now in the helm chart. )
- Multi-cloud and multi-cluster templates to accommodate different cloud providers and Kubernetes distributions.
- Continuous Deployment (CD) processes using ArgoCD or FluxCD as part of future contributions.

I believe these enhancements will greatly benefit the community, and I'm excited to start with the basic configurations. As I continue working on this, I will also explore contributing to the Argo/Flux deployments.

Looking forward to your feedback!

**Comment by xet7** [comment]
@jash2105 

Yes.

Docs have been reorganized:

https://github.com/wekan/wekan/blob/main/docs/Platforms/FOSS/Helm.md

https://github.com/wekan/wekan/blob/main/docs/Platforms/FOSS/Helm2.md

Charts are at this repo, branches main and gh-pages, updated with scripts release.sh (main branch) and after that release2.sh (gh-pages branch):

https://github.com/wekan/charts

**Comment by jash2105** [comment]
@xet7 , But I am trying to add more configs for scalable platform and I would like to contribute the work back. 
So , can I work on that?

**Comment by xet7** [comment]
@jash2105 

Is that something else than just sending some PRs to https://github.com/wekan/wekan and https://github.com/wekan/charts ? Please give more details, what do you mean ?

**Comment by jash2105** [comment]
Yes , I am just thinking of sending some prs for the helm chart and maybe wekan docs if they need to included.

**Comment by xet7** [comment]
Yes, you can work on that. But please do not ask permission for 3rd time, if possible.
